### PR TITLE
Add link python -> python3

### PR DESCRIPTION
**Pull request by Eabramov84**

### CODE REVIEW

File: recipes-devtools/python/python3_%.bbappend [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (oleksiimoisieiev) [inline recipes-devtools/python/python3_%.bbappend:1]: Please remove empty line

Comment (Eabramov84) [reply recipes-devtools/python/python3_%.bbappend:2]: accept. and fixed

**Review by oleksiimoisieiev — APPROVED** [review]

**Comment by al1img** [comment]
LGTM

**Comment by Eabramov84** [comment]
not actual

**Pull request closed without merging**